src/ClientBuilder.js

```diff
@@ -129,6 +131,12 @@ class ClientBuilder {
 		return this;
 	}
 
+	withLicense(license) {
```

Comment:
This option should be plural: 
```
withLicenses(licenses) {
   this.licenses = licenses;
   return this;
}
```

Reply:
(and commented calls to this function should be updated accordingly)

Reply:
We had made the decision in the Go SDK to have the method receive a single license and have the option to be called multiple times. Do we want this to be consistent across all SDKs?